### PR TITLE
In file email_generator.py was added method for creating randon email

### PR DESCRIPTION
**Pull request by ninanichik**
This method is generating random first part in email address.
First part of email may contain uppercase and lowercase letters in random order.
This method we can use in ragistration cases and other cases if it's need.
This method was created with goal not to rewrite each time the email.

### CODE REVIEW

**Comment by alexbeatnik** [comment]
You fogot add 'string' and 'random' to requirements

File: oct/pages/email_generator.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class EmailsGenerator:
+
+    def joining_full_email(self):
+        first_part = str(''.join(random.choice(string.ascii_letters) for x in range(7)))
+        second_part = "@mailinator.com"
+        adding_two_parts = first_part + second_part
+        creating_list = list(adding_two_parts)
+        joined_element = ""
+        joining_two_parts = joined_element.join(creating_list)
+        print(joining_two_parts)
```
Comment (AlekseyDidilov) [inline oct/pages/email_generator.py]: @ninanichik Why do you use print here?

File: oct/pages/email_generator.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import string
+
+
+class EmailsGenerator:
+
+    def joining_full_email(self):
+        first_part = str(''.join(random.choice(string.ascii_letters) for x in range(7)))
+        second_part = "@mailinator.com"
+        adding_two_parts = first_part + second_part
+        creating_list = list(adding_two_parts)
```
Comment (AlekseyDidilov) [inline oct/pages/email_generator.py]: @ninanichik Why do we need lines 11-13?

File: oct/pages/email_generator.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import random
+import string
+
+
+class EmailsGenerator:
+
+    def joining_full_email(self):
```
Comment (bambula4000) [inline oct/pages/email_generator.py]: Please, write what type of data method will return

File: oct/pages/email_generator.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        first_part = str(''.join(random.choice(string.ascii_letters) for x in range(7)))
+        second_part = "@mailinator.com"
```
Comment (bambula4000) [inline oct/pages/email_generator.py]: We don't use variables, if it use only once.

File: oct/pages/email_generator.py (hunk 3 above)
Comment (ninanichik) [reply oct/pages/email_generator.py]: Thanks, fixed

File: oct/pages/email_generator.py (hunk 1 above)
Comment (ninanichik) [reply oct/pages/email_generator.py]: @AlekseyDidilov thanks, print was deleted. Lines 11-13 were created for clarity, now I cut down it

**Comment by ninanichik** [comment]
> You fogot add 'string' and 'random' to requirements
@alexbeatnik it doesn't make sense

File: oct/pages/email_generator.py (hunk 4 above)
Comment (ninanichik) [reply oct/pages/email_generator.py]: @bambula4000 thanks, I have already fixed it

File: oct/pages/creating_emails.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# pylint: disable=no-self-use # pyATS-related exclusion
+import random
+import string
+
+
+class EmailsGeneration:
+    def creating_full_email(self) -> str:
+        first_part = str("".join(random.choice(string.ascii_letters) for x in range(7)))
+        adding_two_parts = list(first_part + "@mailinator.com")
+        joining_two_parts = "".join(adding_two_parts)
```
Comment (GoshkaEgoshka) [inline oct/pages/creating_emails.py]: You can add a return in line 10 without the variable joining_two_parts

Comment (AlekseyDidilov) [reply oct/pages/creating_emails.py]: @ninanichik You will return 8 line if you add "@mailinator.com" here.

File: oct/pages/creating_emails.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# pylint: disable=no-self-use # pyATS-related exclusion
+import random
+import string
+
+
+class EmailsGeneration:
+    def creating_full_email(self) -> str:
+        first_part = str("".join(random.choice(string.ascii_letters) for x in range(7)))
```
Comment (AlekseyDidilov) [inline oct/pages/creating_emails.py]: @ninanichik you don't use "x", please replace it on "_"

**Comment by ninanichik** [comment]
> @ninanichik why do we have this class? Please use it in some test. Also, `oct/pages/` is not the best location for it.

@extsoft Should I remove this class? And /test/web/ is it normal location?

**Pull request merged**